### PR TITLE
Update to Pointing Model sat_v1

### PR DESCRIPTION
**Pull request by elle-shaw**
1. Added fp_offset xi and eta parameters
2. Changed name of bs_xi0 bs_eta0 to fp_rot xi and eta
3. All parameters with units now fit in radians
4. Added capability to fit for Az Rotation

### CODE REVIEW

File: sotodlib/coords/pointing_model.py [hunk 1]
```diff
@@ -146,10 +149,11 @@ def model_sat_v1(params, az, el, roll):
 
     The implemented model parameters are:
 
-      - bs_{xi,eta}0: within the focal plane (i.e. relative to the
+      - fp_rot_{xi,eta}0: within the focal plane (i.e. relative to the
         corrected boresight), the center of rotation of the boresight.
-        Radians.
-      - enc_offset_{az,el,boresight}: encoder offsets, in degrees.
+         Radians.
+      - fp_offset_{xi,eta}0: Offset of focal plane's rotational center relative to position in focal plane that lies along the optical axis. Corrects for "collimation error"
```
Comment (mhasself) [inline sotodlib/coords/pointing_model.py:155]: Wrap docstring lines to 78 chars or so, like elsewhere.  End sentence with a period

File: sotodlib/coords/pointing_model.py [hunk 2]
```diff
@@ -146,10 +149,11 @@ def model_sat_v1(params, az, el, roll):
...
+      - enc_offset_{az,el,boresight}: encoder offsets, in Radians.
       - base_tilt_{cos,sin}: base tilt coefficients, in radians.
```
Comment (mhasself) [inline sotodlib/coords/pointing_model.py:158]: Add az_rot.

File: sotodlib/coords/pointing_model.py [hunk 3]
```diff
@@ -146,10 +149,11 @@ def model_sat_v1(params, az, el, roll):
...
     The implemented model parameters are:
 
-      - bs_{xi,eta}0: within the focal plane (i.e. relative to the
+      - fp_rot_{xi,eta}0: within the focal plane (i.e. relative to the
         corrected boresight), the center of rotation of the boresight.
-        Radians.
-      - enc_offset_{az,el,boresight}: encoder offsets, in degrees.
+         Radians.
+      - fp_offset_{xi,eta}0: Offset of focal plane's rotational center relative to position in focal plane that lies along the optical axis. Corrects for "collimation error"
+      - enc_offset_{az,el,boresight}: encoder offsets, in Radians.
```
Comment (mhasself) [inline sotodlib/coords/pointing_model.py:156]: Might as well state the sign convention here.  Cause I forget which it is...

File: sotodlib/coords/pointing_model.py [hunk 4]
```diff
@@ -167,20 +171,25 @@ def model_sat_v1(params, az, el, roll):
...
+    q_fp_rot = ~quat.rotation_xieta(params['fp_rot_xi0'], params['fp_rot_eta0'])
+    
+    # Rotation that moves the center of the focal plane to fp_offset_(xi, eta)0.  
+    q_fp_offset = quat.rotation_xieta(params['fp_offset_xi0'], params['fp_offset_eta0'])
+
 
     # Horizon coordinates.
     q_hs = (base_tilt * quat.rotation_lonlat(-az, el)
-            * ~q_fp_bs * quat.euler(2, roll) * q_fp_bs)
+             *q_fp_offset * ~q_fp_rot* quat.euler(2, roll) * q_fp_rot)
```
Comment (mhasself) [inline sotodlib/coords/pointing_model.py:192]: add space after first `*` (and align that `*` under the `b` of `base_tilt`).

Add space after `~q_fp_rot`, before `*`.

File: sotodlib/coords/pointing_model.py [hunk 5]
```diff
@@ -216,4 +225,4 @@ def get_base_tilt_q(c, s):
     # And that base tilt causes the true el to lie below the expected
     # (encoder) el, at that position.
     amp = (c**2 + s**2)**.5
-    return quat.euler(2, phi) * quat.euler(1, amp) * quat.euler(2, -phi)
+    return quat.euler(2, phi) * quat.euler(1, amp) * quat.euler(2, -phi)
```
Comment (mhasself) [inline sotodlib/coords/pointing_model.py:228]: Restore newline at end of file.

**Review by mhasself — CHANGES_REQUESTED** [review]
Thanks!  Just some formatting comments.

File: sotodlib/coords/pointing_model.py (hunk 1 above)
Comment (elle-shaw) [reply sotodlib/coords/pointing_model.py:155]: Done.

File: sotodlib/coords/pointing_model.py (hunk 2 above)
Comment (elle-shaw) [reply sotodlib/coords/pointing_model.py:158]: Done: "az_rot: Dimensionless parameter describing a linear dependence of Az on El."

File: sotodlib/coords/pointing_model.py (hunk 3 above)
Comment (elle-shaw) [reply sotodlib/coords/pointing_model.py:156]: Done, added: "Sign convention: True = Encode + Offset"

File: sotodlib/coords/pointing_model.py (hunk 4 above)
Comment (elle-shaw) [reply sotodlib/coords/pointing_model.py:192]: done

**Review by mhasself — APPROVED** [review]
Looks good, thanks!

**Pull request merged**